safe/gui/tools/impact_merge_dialog.py

```diff
@@ -546,6 +556,28 @@ def validate_all_layers(self):
                         'Keyword aggregation attribute not found for '
                         'aggregation layer.'))
 
+    def normalize_dict_keys(self, input_dict):
```

Comment:
Need ``.. versionadded:: 3.2`` in docstrings for new methods and e.g. ``.. version changed:: 3.2 added parameter foo``  for methods whose api has changed